### DISCUSSION TITLE
Gamelauncher not installed?

### DISCUSSION THREAD

**Discussion by EmIsGreat** [prasanthrangan/hyprdots#1863] [Q&A]
I've just installed Hyprdots for the first time, but now I can't find the Gamelauncher. Can somebody help?

**Comment by kRHYME7** [comment]
It's a custom Script 

gamelauncher.sh

**Reply by EmIsGreat** [reply]
Where would I find this script?

**Reply by kRHYME7** [reply]
.local/share/bin
